src/Dump.ts

```diff
@@ -203,7 +203,7 @@ export class Dump {
 
         /* Push Mediawiki:Offline.css (at the end) */
         // TODO: Weak URL (might fail in a number of cases where the wiki path is not like on Wikipedia)
-        const offlineCssUrl = `${downloader.mw.webUrl.href}?title=Mediawiki:offline.css&action=raw`;
+        const offlineCssUrl = downloader.mw.getWebArticleUrlRaw('?title=Mediawiki:offline.css&action=raw');
```

Comment:
This should only take articleId in argument.

Reply:
@kelson42 but there is no other place this is being used better to rename it as getWebQueryUrl ?????